AGENT_INTEGRATIONS.md

```diff
@@ -11,7 +127,7 @@
 * datadog-cassandra-nodetool: 1.3.0
 * datadog-cassandra: 1.3.0
 * datadog-ceph: 1.4.0
-* datadog-checks-base: 6.0.0
+* datadog-checks-base: 6.0.1
```

Comment:
Why is this and things below being updated/deleted?

Reply:
This is now correct with what we have in the agent. The generation task is based on the file `requirements-agent-release.txt`, but this wasn't updated during the last release after some fixes had been merged during QA. 
For the others, the task never accounted for new integrations, which explain the bunch of new additions